src/test/regress/expected/multi_function_in_join.out

```diff
@@ -176,6 +196,8 @@ $$ language plpgsql;
 SELECT * FROM table1 JOIN max_and_min() m ON (m.maximum = data OR m.minimum = data) ORDER BY 1,2,3,4;
 DEBUG:  generating subplan 14_1 for subquery SELECT minimum, maximum FROM functions_in_joins.max_and_min() m(minimum, maximum)
 DEBUG:  Plan 14 query after replacing subqueries and CTEs: SELECT table1.id, table1.data, m.minimum, m.maximum FROM (functions_in_joins.table1 JOIN (SELECT intermediate_result.minimum, intermediate_result.maximum FROM read_intermediate_result('14_1'::text, 'binary'::citus_copy_format) intermediate_result(minimum integer, maximum integer)) m ON (((m.maximum OPERATOR(pg_catalog.=) table1.data) OR (m.minimum OPERATOR(pg_catalog.=) table1.data)))) ORDER BY table1.id, table1.data, m.minimum, m.maximum
+DEBUG:  14_1 is send to localhost:57637
```

Comment:
Should this end with `t` -> `sent`?

Reply:
`14_1 is broadcasted to localhost:57637` sounds better to me. I guess I will update all those 30+ test outputs one more time

Reply:
sorry about that : /

Reply:
@onderkalaci I think we should use `sent` or something similar because `broadcast` means sending to every node, so to me it seems incorrect to use that when we are not actually `broadcasting` that. We are just sending it to some nodes.

Reply:
By no means I'm an english expert, and you might be right. But,  `broadcast` does not necessarily feel `to every`, it feels closer to `spread` to me. And, in the code there is no reference to `send` in this context, so I'd rather use something else.

Given that this is a `DEBUG` message, I don't want to ask updating too many files again. If that's disturbing you, feel free to open a PR on top of this to fix it.

Reply:
I think we can use `multicast`. From https://techdifferences.com/difference-between-broadcast-and-multicast.html :

```
The transmission technologies broadcast and multicast are different from each other in a
context that in the broadcast, the packet is forwarded to all the hosts connected to the
network whereas, in multicast, the packet is forwarded only to intended recipients.
```

I guess we are doing multicasting, so I would suggest to use `multicast`. I can update it if you guys agree

Reply:
I am not really familiar with the word `multicast` but it seems like a better fit in this scenario.

One more idea @SaitTalhaNisanci suggested makes sense as well: using future tense instead of past tense. As this is part of planner, it can be helpful to stress that the data is not actually sent to the worker at the time of logging.

Reply:
We hid new Debug messages behind a GUC `log_intermediate_results`

The final log messages are as follows:

`Subplan %s will be written to local file`
and 
`Subplan %s will be sent to %s:%d`

They will appear only when `the GUC is set, or the min_messages is `DEBUG4` or below